### PR TITLE
Various copy updates

### PR DESCRIPTION
**Pull request by jamesrose**

### CODE REVIEW

File: app/views/registration_wizard/headteacher_duration.html.erb [hunk 1]
```diff
@@ -8,14 +8,17 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <%= form_with model: @form, url: registration_wizard_form_url(@form), scope: 'registration_wizard', method: :patch do |f| %>
-      <%= f.govuk_error_summary %>
 
+      <h1 class="govuk-heading-xl">Are you a headteacher?</h1>
+      <p class="govuk-body">Extra support is available if you are in the first 2 years of your headship.</p>
+
+      <%= f.govuk_error_summary %>
```
Comment (asmega) [inline app/views/registration_wizard/headteacher_duration.html.erb:15]: error summary should be above the `h1` see https://design-system.service.gov.uk/components/error-summary/

File: app/views/registration_wizard/qualified_teacher_check.html.erb [hunk 2]
```diff
@@ -18,22 +18,22 @@
...
       <%= f.govuk_date_field :date_of_birth,
         date_of_birth: true,
-        legend: { text: "Date of birth" },
+        legend: { text: "Date of birth", size: "s" },
         hint: { text: "For example, 31 3 1980" } %>
 
       <%= f.govuk_text_field :national_insurance_number,
-        width: 10,
-        label: { text: "National Insurance number (optional)" },
+        width: "three-quarters",
```
Comment (asmega) [inline app/views/registration_wizard/qualified_teacher_check.html.erb:35]: i don't this is right so i changed it. https://design-system.service.gov.uk/patterns/national-insurance-numbers/ suggests 10

File: spec/features/happy_journeys_spec.rb [hunk 3]
```diff
@@ -227,7 +227,7 @@
     expect(check_answers_page.summary_list["Full name"].value).to eql("John Doe")
     expect(check_answers_page.summary_list["TRN"].value).to eql("1234567890")
     expect(check_answers_page.summary_list["Date of birth"].value).to eql("December 13, 1980")
-    expect(check_answers_page.summary_list["National Insurance number"].value).to eql("AB123456C")
+    expect(check_answers_page.summary_list["National Insurance Number"].value).to eql("AB123456C")
```
Comment (asmega) [inline spec/features/happy_journeys_spec.rb:230]: `Number` should be lower case see https://design-system.service.gov.uk/patterns/national-insurance-numbers/

**Review by asmega — CHANGES_REQUESTED** [review]

File: app/views/registration_wizard/qualified_teacher_check.html.erb (hunk 2 above)
Comment (jamesrose) [reply app/views/registration_wizard/qualified_teacher_check.html.erb]: Updated

**Review by asmega — APPROVED** [review]

**Pull request merged**